extension/Custom/src/DependencyInjection/CustomExtension.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Custom\DependencyInjection;
+
+use Symfony\Component\Config\FileLocator;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+class CustomExtension extends Extension implements PrependExtensionInterface
```

Comment:
Why do we need this extension?

Reply:
This is a custom bundle for placing custom generated Task/TakSets and etc.